sql-cli/conftest.py

```diff
@@ -8,10 +8,10 @@
 from airflow.utils import timezone
 from airflow.utils.session import create_session
 
+from astro.render.workflow_directory_parser import SqlFile, WorkflowFile
```

Comment:
Is the `SqlFile` also used by the implementation in which we create expanded DAGs?